src/transformers/tokenization_utils_base.py

```diff
@@ -1572,8 +1572,8 @@ def _set_processor_class(self, processor_class: str):
 
     def __repr__(self) -> str:
         return (
-            f"{'PreTrainedTokenizerFast' if self.is_fast else 'PreTrainedTokenizer'}(name_or_path='{self.name_or_path}',"
-            f" vocab_size={self.vocab_size}, model_max_len={self.model_max_length}, is_fast={self.is_fast},"
+            f"{'PreTrainedTokenizerFast' if self.is_fast else 'PreTrainedTokenizer'}(name_or_path='{self.__class__.__name__}',"
```

Comment:
I believe what @sgugger means is to change `'PreTrainedTokenizerFast' if self.is_fast else 'PreTrainedTokenizer'` to `self.__class__.__name__`, so we can see the actual class name in the representation.

The `name_or_path` is the local path or remote url of the tokenizer, which should not be changed.

Reply:
Oups I think you are right